src/main/java/uk/gov/pay/adminusers/queue/event/EventMessageHandler.java

```diff
@@ -182,6 +182,17 @@ private Map<String, String> getMinimumRequiredPersonalisation(String gatewayAcco
                 "serviceReference", transaction.getReference()));
     }
 
+    private Map<String, String> getMinimumPersonalisationForDisputeLost(DisputeLostDetails details, String parentResourceExternalId) {
+        var personalisation = getMinimumRequiredPersonalisation(details.getGatewayAccountId(), parentResourceExternalId);
+
+        if (details.getFee() != 0) {
```

Comment:
```suggestion
        if (details.getFee() !=null) {
```

Reply:
Oops.  I failed to spot that this was a 'Long', not a 'long'.